x-pack/plugins/cases/docs/openapi/components/schemas/payload_alert_comment.yaml

```diff
@@ -0,0 +1,21 @@
+type: object
+properties:
+  comment:
+    type: object
+    properties:
+      alertId:
+        type: string
```

Comment:
This can be a string on an array of strings. Same for the index. If both of them are arrays, then the index of `alertId[0]` is in `index[0]`, the index of `alertId[1]` is in `index[1]`, etc.

Reply:
Here we do not document it https://www.elastic.co/guide/en/kibana/8.3/cases-api-add-comment.html

Reply:
Yes, we are leveraging the double array for the bulk adding of alerts. Ok, let's add it then.

Reply:
I've updated the data types in the add comment endpoint via a8a3a1786be3b42a54baf4bdf15886f3613b4cda. Does the same change need to occur in the [update case](https://www.elastic.co/guide/en/kibana/8.3/cases-api-update-comment.html) docs too?

Reply:
Thanks, @lcawl! Yes, the same change needs to occur to the "update comment" docs.

Reply:
Cool, I've added those changes via 27a51f85c5050bea3d688ebfd915ab7afeaa8105